pkg/asset/internal/templates.go

```diff
@@ -145,6 +145,7 @@ spec:
         component: kube-apiserver
       annotations:
         checkpointer.alpha.coreos.com/checkpoint: "true"
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
Given this is a daemonset, should this also get the toleration? or should it not have the annotation at all? The taint is currently only scheduler side (not kubelet enforced) -- daemonsets don't use scheduler -- so my guess is that this would be ignored (unless daemon controller respects the taint?)